src/main/resources/application.yml

```diff
@@ -14,7 +17,6 @@ spring:
     show-sql: true
     properties:
       hibernate:
-        dialect: org.hibernate.dialect.MySQLDialect
```

Comment:
HHH90000025: MySQLDialect does not need to be specified explicitly using 'hibernate.dialect' (remove the property setting and it will be selected by default)
라고 나오네요
특정 Dialect를 사용하려는게 아니면 제거하는게 좋을 것 같네요 👍👍👍